hack/run-wmcb-ci-e2e-test.sh

```diff
@@ -3,57 +3,79 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-RELEASE="release-4.6"
-GIT_CLONE="git clone --branch $RELEASE --single-branch"
 SKIP_VM_SETUP=""
 VM_CREDS=""
 
-while getopts ":v:s" opt; do
-  case ${opt} in
-    v ) # process option for providing existing VM credentials
-      VM_CREDS=$OPTARG
-      ;;
-    s ) # process option for skipping setup in VMs
-      SKIP_VM_SETUP="-skipVMSetup"
-      ;;
-    \? )
-      echo "Usage: $0 [-v] [-s]"
-      exit 0
-      ;;
-  esac
-done
-
-# TODO: Add steps to execute code that will kick off the e2e tests in CI
+# downloads the oc binary but we are just using kubectl as we dont need oc specific commands, oc binary depends on kubectl
+# using only kubectl binary reduces the hassle of moving both oc and kubectl to path. We can use kubectl directly from /tmp
+get_kubectl(){
+  # Download the kubectl binary only if it is not already available
+  # We do not validate the version of kubectl if it is available already
+  if type kubectl >/dev/null 2>&1; then
+    which kubectl
+    return
+  fi
+
+  KUBECTL_DIR=/tmp/kubectl
+  curl -L -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.2.2/openshift-client-linux-4.2.2.tar.gz -o openshift-origin-client-tools.tar.gz \
```

Comment:
Any reason to pick an old version? We should at least use `4.5`.

Reply:
took this snippet from previous dockerfile. Will update to use latest